### ISSUE TITLE
I know it's not and issue but I wanted to ask, could it be possible for UEFormat to be supported by 3DS Max 2025?

### ISSUE THREAD

**Issue by WWIA7062** [halfuwu/UEFormat#5]
Hi, I know it's probably not possible but I just wanted to ask if it was possible for one day for a way to use models and animations exported with the UEFormat to be used in 3DS Max 2025, I'm sorry for bothering you guys about it but I just was curious if it was possible or something to be considered to be added into future support because I actually do use Blender, UE 5.4.1 and 3DS Max 2025 so it'd be awesome if I could use them with all 3 apps.

**Comment by halfuwu** [comment]
oh geez idek, i have never touched 3ds max but i don't think it would be terribly hard to implement

i've got a lot on my plate rn but if anyone wants to take up on trying to implement it then go ahead

**Comment by WWIA7062** [comment]
Yeah, I understand and TBH I don't know the first thing about making scripts for 3DS Max but thanks for letting me know :)

**Comment by WWIA7062** [comment]
I think I should leave this open just in case if someone who knows how to do this stuff for 3DS Max sees this and has interest on importing these formats to 3DS Max. (I doubt that would happen but I don't know but thanks again for letting me know)

**Comment by halfuwu** [comment]
lol when I get the chance I'll check if my student email gives me free 3ds max and then I can give it a shot

**Comment by WWIA7062** [comment]
I just saw your message and I wanna let you know that you should be able to since that's how I got it. I just went to Autodesk's site (through this link) https://www.autodesk.com/education/edu-software/overview?sorting=featured&filters=individual and followed the instructions after clicking get started with my school email and I got access to every Autodesk app when I activate it on that page (you need to verify your school email every year if you wanted to keep it and that's to prevent people like me abusing it and keeping it forever)

**Comment by WWIA7062** [comment]
I just wanted to let you know because if you don't activate your student/educator status and software through that part of the site it won't work and I tried.

**Comment by WWIA7062** [comment]
but if you're busy you don't have to worry about it though :)

**Comment by WWIA7062** [comment]
did you ever get the access to Autodesk software or no? (I'm just curious)

**Comment by AstricX9** [comment]
I also wanted this script to be available for 3ds max. I'm using 3ds Max 2024
